knock/src/components/Profile/Profile.module.scss

```
@@ -0,0 +1,26 @@
+@import '../../core/styles/global.scss';
+
+.container {
+  width: 11.0625rem;
+  height: 15rem;
+  display: inline-flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 0.75rem;
+  justify-content: center;
+}
+
+.profileImage {
+  display: flex;
+  width: 8.5rem;
+  height: 8.5rem;
+  justify-content: center;
+  align-items: center;
+  flex-shrink: 0;
+  padding-left: 1.28rem;
+  padding-right: 1.28rem;
+}
+
+.text {
+  @include H2;
+}
```

컨벤션 설정한 것처럼 하면 더 좋을 것 같아요! 😄
```suggestion
.profile {
  width: 11.0625rem;
  height: 15rem;
  display: inline-flex;
  flex-direction: column;
  align-items: flex-start;
  gap: 0.75rem;
  justify-content: center;


  &__image {
    display: flex;
    width: 8.5rem;
    height: 8.5rem;
    justify-content: center;
    align-items: center;
    flex-shrink: 0;
    padding-left: 1.28rem;
    padding-right: 1.28rem;
  }

  &__name {
    @include H2;
  }
}
```